charts/lupa/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: lupa
+description: Lupa Helm chart for Kubernetes
+type: application
+version: 0.1.0
+appVersion: "v0.0.2"
```

Comment:
Actually, current releases are going to be removed so I would already set it to v0.0.1 (unless its for testing purposes so it can be changed in the future)

Reply:
Done